### ISSUE TITLE
Improve accessibility tests

### ISSUE THREAD

**Issue by phrdang** [phrdang/berkeley-class-site#7]
- [x] Separate axe checks in `spec/accessibility_spec.rb` for each page into separate rspec tests so that the one failure doesn't mean the rest of the pages aren't checked
- [x] Don't hardcode site URL in `spec/accessibility_spec.rb`

**Comment by phrdang** [comment]
Fixed in https://github.com/berkeley-eecs/berkeley-class-site/pull/17

**Comment by phrdang** [comment]
Fixed in https://github.com/berkeley-eecs/berkeley-class-site/pull/17
